views/teams/topics.hbs

```diff
@@ -128,7 +128,7 @@
             {{/userHasPermission}}
             <h2 class="h4">Tools</h2>
 
-            <div class="list-group">
+            <div class="list-group" data-testid="course_tool_list">
```

Comment:
this is actually wrong as we are not in "course" here

Reply:
changed to lti_tool_list